### PR TITLE
disable 'add paused' parameter if trakt rolling is enabled

### PR DESCRIPTION
**Pull request by gborri**
Changed also label accordingly to @fernandog suggestion

### CODE REVIEW

**Comment by fernandog** [comment]
@gborri you can remove the ":" in the labels

File: gui/slick/interfaces/default/config_notifications.tmpl [hunk 1]
```diff
@@ -1544,13 +1551,13 @@
                             <div id="content_trakt_use_rolling_download">
               	                <div class="field-pair">
                                     <label for="trakt_rolling_num_ep">
-                                        <span class="component-title">Number of Episode:</span>
+                                        <span class="component-title">Quantity of episodes:</span>
```
Comment (abeloin) [inline gui/slick/interfaces/default/config_notifications.tmpl]: @gborri @fernandog You don't say "Quantity of episodes" in English, "Number of episodes" is better

**Comment by fernandog** [comment]
@gborri @abeloin ok! sorry!

File: gui/slick/interfaces/default/config_notifications.tmpl [hunk 2]
```diff
@@ -1564,13 +1571,13 @@
...
                                     <label for="trakt_rolling_add_paused">
-                                        <span class="component-title">Should new show to be added paused?:</span>
+                                        <span class="component-title">Add new show as paused:</span>
                                         <span class="component-desc">
                                 	    <input type="checkbox" name="trakt_rolling_add_paused" id="trakt_rolling_add_paused" #if $sickbeard.TRAKT_ROLLING_ADD_PAUSED then "checked=\"checked\"" else ""# />
                                     </label>
                                     <label>
                                         <span class="component-title">&nbsp;</span>
-                                        <span class="component-desc">This feauture will try to snatch <i>number of episode</i> if the show is active. Whould you like to add new show in paused mode(this override previous choice)?</span>
+                                        <span class="component-desc">Stop rolling download to start to download episode.</span>
```
Comment (abeloin) [inline gui/slick/interfaces/default/config_notifications.tmpl]: Prevent rolling download snatching new episode.

**Comment by gborri** [comment]
@fernandog @abeloin done!

**Comment by abeloin** [comment]
@gborri Please squash the changes in 1 commit

Thanks

**Comment by gborri** [comment]
@abeloin i don't know how to do that.

**Comment by abeloin** [comment]
See this: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by gborri** [comment]
@abeloin have tried following the instruction on your url, this is the result......

File: gui/slick/interfaces/default/config_notifications.tmpl [hunk 3]
```diff
@@ -1514,17 +1514,24 @@
                                 </div>
                                 <div class="field-pair">
                                     <label for="trakt_start_paused">
-                                        <span class="component-title">Start paused:</span>
+                                        <span class="component-title">Start paused</span>
+#if not $sickbeard.TRAKT_USE_ROLLING_DOWNLOAD
```
Comment (fernandog) [inline gui/slick/interfaces/default/config_notifications.tmpl]: @gborri please align this

**Comment by fernandog** [comment]
@gborri also squash commits into one

**Pull request closed without merging**